pages/language/[tag].tsx

```diff
@@ -19,7 +21,7 @@ export default function Language() {
         <title>{pageTitle}</title>
       </Head>
       <RepositoryList
-        repositories={repositories.filter((repository) => repository.language.id == tag)}
+        repositories={repositories.filter((repository) => repository.language.id == tag)} dtOrder={data.date} starOrder={data.star}
```

Comment:
With the sorting strategy in the context, we won't need to pass it around in every place anymore.